daliuge-engine/dlg/apps/branch.py

```diff
@@ -9,7 +9,7 @@
 # @par EAGLE_START
 # @param category Branch
 # @param tag template
-# @param dropclass dlg.apps.simple.SimpleBranch/String/ComponentParameter/NoPort/ReadOnly//False/False/Application class
+# @param dropclass dlg.apps.branch.BranchAppDrop/String/ComponentParameter/NoPort/ReadOnly//False/False/Application class
 # @param base_name simple/String/ComponentParameter/NoPort/ReadOnly//False/False/Base name of application class
```

Comment:
Do we still need the BranchAppDrop when we have the new Branch(PyFuncApp)

Reply:
Only for the existing graphs and tests.